src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -372,8 +372,8 @@ std::string
 PeerImp::getVersion() const
 {
     if (inbound_)
-        return headers_["User-Agent"];
-    return headers_["Server"];
+        return std::string{headers_["User-Agent"]};
+    return std::string{headers_["Server"]};
```

Comment:
okay. But I don't think that will provide us with any performance gains, because we need to convert to `std::string` in order to insert them into `Json::Value` objects. So, there is one extra copy performed somewhere along the stack.

Reply:
fixed in [9d9f40e](https://github.com/XRPLF/rippled/pull/4509/commits/9d9f40e6cb0f5e1342fe9c5e156258e72b15da6a)